webapp/src/test/resources/integration-test-data.sql

```diff
@@ -1,13 +1,16 @@
+insert into import (id, status, content, contentType, digest) values
+  (-1, 1, 1, '', '');
```

Comment:
I think I might have been the first one to use this script in RDW_Admin?  A lot of the scripts were broken after recent migrate changes so I updated them and now using this in my new IT